rangeDS/fenwickTree.cpp

```diff
@@ -1,6 +1,16 @@
 struct fenwickTree {
 	vector<ll> bit;
 	fenwickTree(int n) : bit(n,0) {}
+	fenwickTree(const vector<int> & a) : bit(a.size()) {
+		bit[0] = a[0];
```

Comment:
This will RTE if a is empty, let's not have this behavior

Reply:
constructing a vector of size 0 will always RTE, I'm pretty sure...